segmentation_labeling_app/rois/rois.py

```diff
@@ -24,10 +25,112 @@
 sql_insert_roi = """ INSERT INTO rois_prelabeling (transform_hash,
                      ophys_segmentation_commit_hash,
                      creation_date, upload_date, manifest, experiment_id,
-                     roi_id) 
+                     roi_id)
                      VALUES (?, ?, ?, ?, ?, ?, ?) """
 
 
+def cumulative_fraction_threshold(data, target_fraction, nbins=200):
+    """return a threshold value above which the summed weights are
+    target_fraction of the total summed weights
+
```

Comment:
Just for clarification this function returns a value such that the percent of values above the found threshold is equal to the specified target_fraction? Am I understanding that right?

Reply:
I think this part of the docstring is unambiguous:
```
    Returns
    -------
    fval : float
        data[data > fval].sum() / data.sum() ~ target_fraction
```

Reply:
I agree, it was for my own clarification